chase/frontend/components/speakers_list/queue_list.tsx

```diff
@@ -57,6 +82,18 @@ function CountryCard({
           {getCountryNameByCode(countryCode, locale)}
         </div>
       </div>
+      <div className="flex-1" />
+      {chairOptions && (
+        <>
+          {/* TODO Find intuitive way to change the oder of the list */}
```

Comment:
maybe a drag & drop handle would do the job?

Like this https://dribbble.com/shots/19853934--P-0-1

Reply:
Yes, that was my Idea too but didn't finde an elegant option to implement it. I also wanted to wait for the backend to discuss with you how to handle such re-ordering requests. Is there a best practise? After that we can talk about implementation. I would keep the TODO for now